pkg/util/priority/priority.go

```diff
@@ -36,32 +37,60 @@ func Priority(w *kueue.Workload) int32 {
 	return ptr.Deref(w.Spec.Priority, constants.DefaultPriority)
 }
 
+// GetPriorityFromPriorityClassAndSource returns the priority from priorityClassName and source.
+func GetPriorityFromPriorityClassAndSource(ctx context.Context, client client.Client, priorityClassName string, source string) (int32, error) {
```

Comment:
Why add this function? I only see it being used in tests.

Reply:
I am using this function in workload_controller.go.
However, classname and source won't be mutable, so I will remove this function.